corehq/apps/api/object_fetch_api.py

```diff
@@ -128,13 +128,13 @@ def get(self, request, domain, form_id=None, attachment_id=None):
             raise Http404
 
         # this raises a PermissionDenied error if necessary
-        _get_location_safe_form(domain, request.couch_user, form_id)
+        safely_get_form(request, domain, form_id)
```

Comment:
This is an odd function name. I would expect it to return a value. Is that being ignored here? Would it make sense to rename this function, maybe just in this context? Something like
```py
from corehq.apps.reports.views import (
    ...,
    safely_get_form as check_form_permission,
    ...
)
```

Reply:
Yeah, it does return a value.  This usage is a little strange, looks like it wants the entire body of `safely_get_form` to happen (ie permission and existence checks), but it doesn't actually need the form.

I'm not a fan of the rename, because it would obscure what's happening, and if in the future, this needed access the form, I'd want it to be evident that it's already there.  I suppose I could pull in the body of `safely_get_form`, which would make it evident why this is needed (and obviate the comment) but that's introducing duplication.  I'm inclined to leave it, but could be persuaded otherwise.  What do you think?

Reply:
> I'm not a fan of the rename, because it would obscure what's happening, and if in the future, this needed access the form, I'd want it to be evident that it's already there.

Interesting dilemma there, although that justification sounds a bit like prioritizing a future you don't know will come to pass above making the current implementation as readable as possible. I also agree that duplicating part of the function body here would not be a good solution.

I don't feel terribly strongly about it, so I'm fine with leaving it as is.